docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -2538,6 +2693,12 @@ definitions:
             stream_id:
               type: string
               description: the id of the stream that consumes data from this partition
+            assign_type:
```

Comment:
Did you mean `assignment_type`?

Reply:
Good point. Fixed https://github.com/zalando/nakadi/pull/834/commits/c02b2d1d92264a318695c6bcb2aa90c610d98f43